docs/architecture.md

```diff
@@ -106,15 +103,19 @@ You can enqueue a workflow from within a DBOS app directly or from anywhere usin
 
 When you enqueue a workflow, it may be executed on any of your application's servers.
 All DBOS applications periodically poll their queues to find and execute new work.
-This is in contrast to other queue services that have separate "worker servers" that can execute queued tasks.
-In DBOS, all of your application servers act as queue workers, as in this diagram:
+Essentially, all of your application servers act as queue workers, as in this diagram:
 
 <img src={require('@site/static/img/architecture/dbos-queues.png').default} alt="DBOS Queues" width="750" className="custom-img"/>
 
+Sometimes, you want to separate the worker servers that execute your queued tasks from the rest of your application.
```

Comment:
It's still buried here. I wonder if we should start a new tutorial page saying "distributed worker/server deployment"?